sr_port/db_auto_upgrade.c

```diff
@@ -203,6 +203,10 @@ void db_auto_upgrade(gd_region *reg)
 				csd->read_only = 0;
 				break;
```

Comment:
Should this break be removed so we can fall into the next section?

Reply:
Good point. Now done.